src/code/utilities/trial-generator.js

```diff
@@ -34,6 +51,16 @@ export function generateTrialDrakes(trialDef, trial=0) {
   return [];
 }
 
+function arePhenotypesEqual(drake1, drake2) {
```

Comment:
There's an easier way to do this: `drake1.getImageName() === drake2.getImageName()`.

Image names are by definition the phenotype.